### DISCUSSION TITLE
Keymap with ctrl-shift-c

### DISCUSSION THREAD

**Discussion by peter-jerry-ye** [helix-editor/helix#5143] [Q&A]
Hi,

I'm using helix in WSL. The Windows Terminal uses the Ctrl+Shift+C to copy things to clipboard, but in helix, it seems that it's interpreted as Ctrl+C which comments the chosen part. Is that the expected behaviour?

Thanks.

**Comment by archseer** [comment]
Helix uses `y`/`space y` to yank.

**Reply by peter-jerry-ye** [reply]
I know. But I expect to use the feature of Windows Terminal directly so that I don't have to install extra packages just for yanking.

**Reply by archseer** [reply]
`space y` will yank straight into windows clipboard.

**Reply by archseer** [reply]
`Ctrl+Shift+C`/`Ctrl-C` is just an input. It's up to the application to interpret it.

**Reply by peter-jerry-ye** [reply]
> `space` y will yank straight into windows clipboard.

I tested with helix 22.08.1 and it didn't work, which expected because `helix --health` indicates
```
Clipboard provider: none
System clipboard provider: Not installed
```

> `Ctrl+Shift+C`/`Ctrl-C` is just an input. It's up to the application to interpret it.

I assume the helix is the application we are talking about here? I mean, if I use `Ctrl+Shift+C` in `nano` or in `bash` or in `top`, I can use `Ctrl+Shift+C` directly because these applications do not consume this input and it is able to arrive at the Windows Terminal. In helix, however, this input is consumed and interpreted as `Ctrl+C`. So my question is if there's a way to avoid helix consume this input (since this does not correspond to any defined key bindings).

**Reply by peter-jerry-ye** [reply]
Helix 22.12.0 now has

> Clipboard provider: termcode
> System clipboard provider: termcode

and the `space y` works.
